scripts/openstack.py

```diff
@@ -296,6 +292,90 @@ def get_parser():
         ],
         conflict_handler='resolve',
         add_help=False,
+        epilog="""test repos:
+
+Test repository can be specified using --test-repo optional argument
+with value in the following formats:  NAME:URL, NAME!PRIORITY:URL
+or @FILENAME. See examples:
+
+1) Essential usage requires to provide repo name and url:
+
+    --test-repo foo:http://example.com/repo/foo
+
+2) Repo can be prioritized by adding a number after '!' symbol
+   in the name:
+
+    --test-repo bar!10:http://example.com/repo/bar
+
+3) Repo data can be taken from a file by simply adding '@' symbol
+   at the beginning argument value, for example from yaml:
+
+    --test-repo @path/to/foo.yaml
+
+  where `foo.yaml` content is:
+
+  - name: foo
+    priority: 1
+    url: http://example.com/repo/foo
```

Comment:
Can there be multiple items in the `foo.yaml` file? If so, would it be a good idea to mention that - e.g. by saying 

```
where foo.yaml contains one or more stanzas like:
```

Reply:
actually there is a line above saying this:
```
you can provide several repos within one yaml or json file
```

Reply:
Right. I added this to the above comment: (Later on you clarify this, but please consider the suggested wording?)

The wording I suggested removes doubt from the reader's mind:

```
where foo.yaml contains one or more stanzas like:
```

This nicely prepares the reader for the more detailed explanation that follows.